### ISSUE TITLE
Could not install packages due to an OSError: [WinError 5] Access is denied..

### ISSUE THREAD

**Issue by jhj0517** [jhj0517/stable-diffusion-webui-Layer-Divider#1]
```
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied.. 
'C:\\path\\to\\SDWebUI\\venv\\Lib\\site-packages\\google\\~rotobuf\\internal\\_api_implementation.cp310-win_amd64.pyd'
Check the permissions.
```

During the initial installation of this extension, the above error occurs. 

However, if you completely close the stable-diffusion WebUI and restart it, the extension appears and works fine.

I've investigated this, and it appears that this issue occurs in several extensions, not just my extension.

I'll leave this issue open until I know why this happens and find a solution.

- Related issue
[ sd_dreambooth_extension #813](https://github.com/d8ahazard/sd_dreambooth_extension/issues/813)

**Comment by BrandonM01** [comment]
> fixed at [# 3443daa](https://github.com/jhj0517/stable-diffusion-webui-Layer-Divider/commit/3443daadcbb9a5ddd7557c595e1976de60508276)

I'm new to python, how how did you fix your issue?

**Comment by jhj0517** [comment]
Hi @BrandonM01, If you encounter an error while launching the WebUI with this extension, 
It might be related to #22, fixed in #23.
I've tested on the latest version of the WebUI and it worked fine.

If you still face permission-related error with latest commit of this extension, please let me know.

>> how did you fix your issue?

About it, I honestly don't remember exactly why and didn't even raise the issue with details, but I think the `run` method (the custom module method of AUTOMATIC1111) was problematic, so it worked after I changed it.
